ragna/deploy/_ui/central_view.py

```diff
@@ -138,15 +164,17 @@ def __init__(
             ui.stylesheets(
                 (
                     f":host(.{css_class})",
-                    {"background-color": "rgb(243, 243, 243) !important"}
-                    if role == "user"
-                    else {
-                        "background-color": "none",
-                        "border": "rgb(234, 234, 234)",
-                        "border-style": "solid",
-                        "border-width": "1.2px",
-                        "border-radius": "5px",
-                    },
+                    (
```

Comment:
This will create merge conflicts with #312 that we'll merge soon. But I think this is only formatting anyway so you can just accept whatever we might have upstream.

Reply:
I went ahead and fixed them.